src/components/TeacherViews/Onboarding/Onboarding.view.tsx

```diff
@@ -98,96 +66,48 @@ export const OnboardingView = () => {
         }
       >
         <Logo height={48} width={104} variant="with text" />
-        <OakHeading tag="h2" $font="heading-light-5">
-          Select your school
+        <OakHeading
+          $mb={"space-between-s"}
+          $mt={"space-between-m"}
+          tag="h2"
+          $font="heading-light-6"
+        >
+          Do you work in a school?
         </OakHeading>
+        <OakBox $pv={"inner-padding-xl"}>
+          <OakRadioGroup
+            onChange={(value) =>
+              setWorksInSchool(value.target.value === "yes" ? true : false)
+            }
+            $flexDirection={"column"}
+            name={"Do you work in a school?"}
+          >
+            <OakRadioButton id="option-1" label="Yes" value="yes" />
+            <OakRadioButton id="option-2" label="No" value="no" />
+          </OakRadioGroup>
+        </OakBox>
 
-        <ResourcePageSchoolPicker
-          hasError={formState.errors?.school !== undefined}
-          schoolPickerInputValue={schoolPickerInputValue}
-          setSchoolPickerInputValue={onSchoolPickerInputChange}
-          schools={schools}
-          label={"School"}
-          setSelectedSchool={setSelectedSchool}
-          required={true}
-          withHomeschool={false}
-        />
         <OakPrimaryButton
+          $mv={"space-between-m"}
           disabled={
-            formState.errors?.school !== undefined || !formState.isValid
+            formState.errors?.worksInSchool !== undefined || !formState.isValid
+          }
+          element={
+            formState.errors?.worksInSchool !== undefined || !formState.isValid
+              ? "button"
```

Comment:
how come it changes element type?

Reply:
a tag has no 'disabled' state. So this switches to button so we can have that prop.